src/core/modules/StateEvaluator.ts

```diff
@@ -205,6 +208,8 @@ export interface EvaluationOptions {
   // an endpoint for retrieving wallet balance info
   walletBalanceUrl: string;
 
+  sourceType: SourceType;
```

Comment:
please add a comment what does this option represent.

Reply:
done